aspnetcore/tutorials/dotnet-watch.md

```diff
@@ -75,10 +75,18 @@ Any [.NET Core CLI command](/dotnet/core/tools#cli-commands) can be run with `do
 
 Run `dotnet watch run` in the *WebApp* folder. The console output indicates `watch` has started.
 
-::: moniker range=">= aspnetcore-3.1"
-Starting with the .NET Core 5.0 SDK, invoking `dotnet watch run` on a webapp launches a browser that navigates to the app's url once ready. `dotnet watch` does this by reading the app's console output and waiting for the the ready message printed by <xref:Microsoft.AspNetCore.WebHost>. 
+::: moniker range=">= aspnetcore-5.0"
```

Comment:
I changed this to 5 from >= 3.1

Reply:
Fair enough. Technically this works on 3.1 projects, but I'm ok not advertising it.